lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/AttributeSetBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.devinci.lib.robolectric;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
```

Comment:
This is android related code, so we can add support annotations and use NonNull?